components/scream/src/physics/shoc/shoc_isotropic_ts_impl.hpp

```diff
@@ -0,0 +1,61 @@
+#ifndef SHOC_ISOTROPIC_TS_IMPL_HPP
+#define SHOC_ISOTROPIC_TS_IMPL_HPP
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+
+namespace scream {
+namespace shoc {
+
+/*
+ * Implementation of shoc isotropic_ts. Clients should NOT
+ * #include this file, but include shoc_functions.hpp instead.
+ */
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::isotropic_ts(
+  const MemberType&            team,
+  const Int&                   nlev,
+  const Int&                   shcol,
+  const Scalar&                brunt_int,
+  const uview_1d<const Spack>& tke,
+  const uview_1d<const Spack>& a_diss,
+  const uview_1d<const Spack>& brunt,
+  const uview_1d<Spack>&       isotropy)
+{
+
+  //constants from physics/share
+  static constexpr  auto ggr = C::gravit;
+
+  //Declate constants as Spacks to use ekat's min and max functions
```

Comment:
This should be looked at carefully. I declared them as Spacks as I wanted to use ekat's min and max functions. I can modify the code if there is a better way to do it.